docs/releases/3.0.3.txt

```diff
@@ -45,3 +45,6 @@ Bugfixes
 * Fixed a regression in Django 3.0 that caused a crash of
   ``QuerySet.prefetch_related()`` for ``GenericForeignKey`` with a custom
   ``ContentType`` foreign key (:ticket:`31190`).
+
+* Fixed a regression in Django 3.0 that caused a file response using a
+  temporary file to be closed incorrectly (:ticket:`31240`).
```

Comment:
Should this be moved to 3.0.4?

Reply:
Great catch :1st_place_medal:

Reply:
Ah, yes. Good one Simon. Getting late. Doh.

Reply:
Done. Doh.